net-libs/msgraph/msgraph-0.2.3.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org meson
+
+DESCRIPTION="GLib-based library for accessing online serive APIs using MS Graph protocol."
+HOMEPAGE="https://gitlab.gnome.org/GNOME/msgraph"
+
+LICENSE="LGPL-2+"
+SLOT="0/1"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc64 ~riscv ~sparc ~x86"
```

Comment:
In which arches were you able to test this? I guess only amd64, in that case, we would need to drop all the other KEYWORDS and open a keywording bug. One option would be to split msgraph/uhttpmock PR, I push it, open the bug, and in some days we will see if we need to drop the keywords or mask onedrive USE flag for gvfs